crap-advisor-api/src/main/kotlin/com/aiecel/crapadvisor/service/ReviewService.kt

```diff
@@ -20,7 +20,11 @@ class ReviewService(
 
     private val log = KotlinLogging.logger { }
 
-    fun getByRestroomId(restroomId: Long) = reviewRepository.findAllByRestroomId(restroomId)
+    fun getByRestroomId(restroomId: Long): List<Review> {
+        val reviews = reviewRepository.findAllByRestroomId(restroomId)
+        log.debug("Fetched ${reviews.size} review(s) by restroom id $restroomId")
```

Comment:
You are really need this logs in master branch?

Reply:
Man logs are very important, how can you see what the backend is doing when it's not writing in the logs? 
Yes, this stays

Reply:
Sounds convincing! I'll resolve this tread now